.isort.cfg

```diff
@@ -0,0 +1,6 @@
+[settings]
```

Comment:
Thanks, I will remove this file. I hope that kwalitee will soon use `isort` instead of `flake8-import-order` as I got import order errors with the configuration generated by cookiecutter. This configuration was the closest I could get to what kwalitee asks, and it still needed some modifications by hand.